test/Conversion/TorchToTosa/linear.mlir

```diff
@@ -0,0 +1,74 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-tosa -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
Rather than adding lit tests, preference in the torch-mlir project is to use e2e testing. 
The file e2e_testing/xfail_sets.py has the TOSA_PASS_SET defined in it, which is the working e2e tests for the TOSA backend. 

Looking at the test suite, (python/torch_mlir_e2e_test/test_suite), the ConvolutionModule2DGroups_basic should test the new functionality you've added. You should only need to enable this test, and then can verify that it passes. Presumably it fails without your change.

Once the e2e test is working, this file is no longer needed.

Reply:
I've added tests for depthwise and grouped convolutions with statically shaped weights and updated `xfail_sets.py` accordingly.